instructure/canvas-android
# [MBL-16351][Student][Teacher] Change the theme (dark/light) only for the RCE content.

Test plan:
- Verify that the theme switch button appears above the html content if the app is in dark mode and the html is not empty.
- There can be cases where html content can be empty (no description for assignment), in these cases the button should not be displayed.
- Verify that the button is not displayed if the app is in light mode.
- Verify that the button is not displayed on screens that are loading a url into the WebView (for example Quiz), it should only show on screens where we load html content.
- Verify that tapping the button changes the html content to the correct color.
- Screens to check:
  - Assignment Details
  - Syllabus
  - Calendar events from syllabus list
  - Discussion details (old discussions, redesign should not display the button)
  - Text Submission Details
  - Quiz Details (only Teacher)
  - Quiz Details in module progression (only Student)
  - K5 Homeroom announcement (only Student)
  - K5 Important links (only Student)

refs: MBL-16351
affects: Student, Teacher
release note: Added a button to switch between dark/light theme in html content.

## Checklist
